views/confirmation.ejs

```diff
@@ -2,105 +2,124 @@
-          data-for="get-referral-phone-1"
-          data-error="invalid-number-error-1"
-          type="tel"
-          value="">
-        <label for="get-referral-phone-1">Phone Number</label>
-      </div>
-      <button class="get-referral-info-btn"
-        data-for="get-referral-phone-1"
-        data-error="invalid-number-error-1">Check</button>
+    <div class="content">
+
+      <p>
+        <% if (fromReferral || fromPartner) { %>
+          Got more friends who would appreciate Shine?
+          <br /> Spread some more motiv-affirmation!
+          <% } else if (fromCampaign) { %>
+            <%= bodyCopy %>
+              <br /> Learn how to get an invite to our
+              <a href="/squad">VIP group</a> and earn free
+              <a href="/referrals">swag</a>.
+              <% } else { %>
+                Want to make it even more official?
+                <br /> Learn how to get an invite to our
+                <a href="/squad">VIP group</a> and earn free
+                <a href="/referrals">swag</a>.
+                <% } %>
+      </p>
+
+      <% if (referralCode.length > 0) { %>
+
+        <section class="referral-container">
+          <h2>Step 1. Copy your referral link:</h2>
+          <div class="referral-link-container" data-clipboard-text="www.shinetext.com/?r=<%= referralCode %>">
+            <input id="referral-link-url" value="www.shinetext.com/?r=<%= referralCode %>" disabled>
+            <button id="copy-referral-link" data-clipboard-text="www.shinetext.com/?r=<%= referralCode %>">Copy Link</button>
+          </div>
+        </section>
+
+        <section class="social-share">
+          <ul class="social-share-icons">
+            <li class="twitter-container">
+              <a href="<%= shareUrls.twitter ? shareUrls.twitter : '#' %>" target="_blank">
+                <div class="twitter icon"></div>
+              </a>
+              <label>Tweet</label>
+            </li>
+            <li class="facebook-container">
+              <a href="<%= shareUrls.facebook ? shareUrls.facebook : '#' %>" target="_blank">
+                <div class="facebook icon"></div>
+              </a>
+              <label>Share</label>
+            </li>
+            <li class="email-container">
+              <a href="<%= shareUrls.email ? shareUrls.email : '#' %>" target="_blank">
+                <div class="email icon"></div>
+              </a>
+              <label>Email</label>
+            </li>
+            <li class="sms-container">
+              <a href="<%= shareUrls.sms ? shareUrls.sms : '#' %>" target="_blank">
+                <div class="sms icon"></div>
+              </a>
+              <label>Text</label>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>Step 2. Earn swag when your friends join</h2>
+          <%- partial ('partials/swag-block.ejs') %>
+        </section>
+        <section>
+          <h2 style="padding-bottom: 20px;">Your referral count</h2>
```

Comment:
To follow the design, let's capitalize the first letters on each word ("Your Referral Count")